### PR TITLE
Feature/repl dump output to file

### PR DESCRIPTION
**Pull request by vladciobanu**
###### Reviewer checklist

- [ ] Test coverage: `stack test --coverage`
- [ ] Public API documentation: `stack haddock`
- [ ] Style conformance: `stylish-haskell`

---

### CODE REVIEW

File: kore/src/Kore/Repl/Interpreter.hs [hunk 1]
```diff
@@ -75,39 +77,48 @@ import qualified Kore.Step.Strategy as Strategy
 import           Kore.Unparser
                  ( unparseToString )
 
+type ReplM level a = RWST () String (ReplState level) Simplifier a
```
Comment (ttuegel) [inline kore/src/Kore/Repl/Interpreter.hs]: The writer part of `RWST` leaks space just like `WriterT`. We should never use it.

Comment (vladciobanu) [reply kore/src/Kore/Repl/Interpreter.hs]: I knew something to that effect, but I thought it's fine for brief uses? We're concatenating a couple of strings before resolving the `RWST` to a `StateT` on every command. Its use is confined to this module, and its lifetime to a single command. Would that still be a problem?

Comment (ttuegel) [reply kore/src/Kore/Repl/Interpreter.hs]: > Its use is confined to this module, and its lifetime to a single command.

Could you explain how that works more? It looks like several commands have acquired `MonadWriter` constraints.

**Comment by ttuegel** [comment]
Approved; we addressed the comments during stand-up.

**Pull request merged**